### PR TITLE
Support flutter version `3.16.0` 

### PR DESCRIPTION
**Pull request by aryan-more**
Added Latest commit of enough_html_editor to support flutter 3.16.0

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -8,7 +8,8 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
-  enough_platform_widgets: 0.6.0
+  enough_platform_widgets:
+    git: https://github.com/Enough-Software/enough_platform_widgets
```
Comment (hoangdat) [inline pubspec.yaml:12]: Why we need to apply it? Why not an official version?

Comment (dab246) [reply pubspec.yaml:12]: @aryan-more  Please explain help us. Thanks

Comment (aryan-more) [reply pubspec.yaml:12]: At the time of upgrading dependency there was error in enough_platform_widgets and the latest changes aren't synced/published on pub.dev.That's why i changed the source of dependency.But as of today they have published latest version with this fixes

Comment (aryan-more) [reply pubspec.yaml]: I have changed the source of enough_platform_widgets from direct git to pub.dev

Comment (dab246) [reply pubspec.yaml]: Ok. @aryan-more  Please rebase it

Comment (aryan-more) [reply pubspec.yaml]: hey @dab246 i have already done the re base

**Comment by dab246** [comment]
@aryan-more Thanks your PR, please squash & rebase to we can merge it

Screenshot 2023-11-27 at 11 01 58

**Pull request merged**